renpy.el

```diff
@@ -2128,11 +2141,9 @@ it contain spaces."
 ;;;###autoload
 (define-derived-mode renpy-mode prog-mode "Ren'Py"
   "Major mode for editing Renpy files.
-Turns on Font Lock mode unconditionally since it is currently required
-for correct parsing of the source.
-See also `jython-mode', which is actually invoked if the buffer appears to
-contain Jython code.  See also `run-renpy' and associated Renpy mode
-commands for running Renpy under Emacs.
+
+See `renpy-run' and associated Renpy mode commands for running Renpy
+under Emacs.
```

Comment:
Just to keep the naming consistent, both instances of "Renpy" should probably be "Ren'Py" here too.